### PR TITLE
Fix casing CollectionType

### LINKED ISSUE #5332: When adding a Movies library in version 10.7 rc4, jellyfin recognizes it as if it were an OTHERS category.
**Describe the bug**
When trying to add a library of Movies or SERIES, it only recognizes it as OTHERS.
The same happens with the other categories. RECOGNIZES THEM AS IF IT WERE OTHERS

**System (please complete the following information):**
 - OS: [Windows 10]
 - Jellyfin Version: [10.7.0, rc4]
 
**Screenshots**
https://imgur.com/aNNlQXS

### PR DESCRIPTION
**Pull request by Bond-009**
Should fix #5332

### CODE REVIEW

**Comment by crobibero** [comment]
The full fix also needs `Path.GetExtension(file)` -> `Path.GetFileNameWithoutExtension(file)` [here](https://github.com/jellyfin/jellyfin/blob/480dd66428e2a8bf76c0747256f2e99b15aae212/Emby.Server.Implementations/Library/LibraryManager.cs#L1250)

With the fix mentioned above I get the following correct result
image

**Comment by Bond-009** [comment]
@crobibero looks like I forgot to `git add` that :facepalm:

**Review by cvium — APPROVED** [review]
Seems to work

File: MediaBrowser.Model/Entities/JsonLowerCaseConverter.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        /// <inheritdoc />
+        public override T Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            return JsonSerializer.Deserialize<T>(ref reader, options);
+        }
+
+        /// <inheritdoc />
+        public override void Write(Utf8JsonWriter writer, T value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value?.ToString().ToLowerInvariant());
```
Comment (BaronGreenback) [inline MediaBrowser.Model/Entities/JsonLowerCaseConverter.cs:26]: Just checking - you want NULL written and not string.empty ?

**Review by BaronGreenback — APPROVED** [review]

**Pull request merged**